Public/Src/Engine/Processes/SandboxedProcessPipExecutor.cs

```diff
@@ -1763,26 +1771,31 @@ private bool PrepareFileAccessMonitoringCommon(LoggingContext loggingContext)
             }
 
             // Untrack the globally untracked paths specified in the configuration
-            foreach (var path in m_sandboxConfig.GlobalUnsafeUntrackedScopes)
+            if (m_pip.RequireGlobalDependencies)
             {
-                // Translate the path and untrack the translated one
-                if (m_fileAccessManifest.DirectoryTranslator != null)
+                foreach (var path in m_sandboxConfig.GlobalUnsafeUntrackedScopes)
                 {
-                    var pathString = path.ToString(m_pathTable);
-                    var translatedPathString = m_fileAccessManifest.DirectoryTranslator.Translate(pathString);
-                    var translatedPath = AbsolutePath.Create(m_pathTable, translatedPathString);
-
-                    if (path != translatedPath)
+                    // Translate the path and untrack the translated one
+                    if (m_fileAccessManifest.DirectoryTranslator != null)
                     {
-                        AddUntrackedScopeToManifest(translatedPath);
-                        Tracing.Logger.Log.TranslatePathInGlobalUnsafeUntrackedScopes(loggingContext, m_pip.SemiStableHash, m_pip.GetDescription(m_context), pathString);
+                        var pathString = path.ToString(m_pathTable);
+                        var translatedPathString = m_fileAccessManifest.DirectoryTranslator.Translate(pathString);
+                        var translatedPath = AbsolutePath.Create(m_pathTable, translatedPathString);
+
+                        if (path != translatedPath)
+                        {
+                            AddUntrackedScopeToManifest(translatedPath);
+                            Tracing.Logger.Log.TranslatePathInGlobalUnsafeUntrackedScopes(m_loggingContext, m_pip.SemiStableHash, m_pip.GetDescription(m_context), pathString);
+                        }
                     }
-                }
 
-                // Untrack the original path
-                AddUntrackedScopeToManifest(path);
+                    // Untrack the original path
+                    AddUntrackedScopeToManifest(path);
+                }
+                Tracing.Logger.Log.PipProcessPulledGlobalUnsafeUntrackedScopes(m_loggingContext, m_pip.SemiStableHash, m_pip.GetDescription(m_context));

```

Comment:
>PipProcessPulledGlobalUnsafeUntrackedScopes [](start = 35, length = 43)

I'm just afraid that there is not much information one can get from this log because he/she can simply look into the pip spec or the pip dump.